### PR TITLE
User Accounts

### PR DESCRIPTION
**Pull request by peterp**
Modals required:
1. Personal details
2. Change password
3. Permissions

### CODE REVIEW

File: go/account/views.py [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
 
 from go.account.forms import EmailForm, AccountForm
 from go.account.tasks import update_account_details
 from go.base.django_token_manager import DjangoTokenManager
 
+from django.contrib.auth.forms import PasswordChangeForm
+
 
-@login_required
 def index(request):
```
Comment (justinvdm) [inline go/account/views.py]: Do we still use `index` anywhere?

File: go/urls.py [hunk 2]
```diff
@@ -19,11 +16,7 @@ def health(request):
 
     # django-regisration auth, override some views so we can specify
     # our own custom forms to use.
```
Comment (justinvdm) [inline go/urls.py]: Is this comment still relevant?

File: go/account/views.py (hunk 1 above)
Comment (peterp) [inline go/account/views.py]: I don't think so, I'll remove it.

File: go/urls.py (hunk 2 above)
Comment (peterp) [inline go/urls.py]: Nope. Removed.

File: go/templates/registration/login.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 {% load url from future %}
 
 {% block auth_form %}
-<form action="{% url 'auth_login' %}" method="post">
+<form action="" method="post">
```
Comment (justinvdm) [inline go/templates/registration/login.html:5]: This is probably me not understanding django or `django.contrib.auth` properly, but why are we removing `{% url 'auth_login' %}`?

Comment (peterp) [inline go/templates/registration/login.html:5]: `action=""` submits the form to the same url, and in this instance the post and get actions of this form are at the same endpoint. So, they're essentially the same thing.

Comment (justinvdm) [inline go/templates/registration/login.html:5]: Ah, ok cool.

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**